apps/mux/src/locations/config.tsx

```diff
@@ -209,7 +227,7 @@ class Config extends React.Component<ConfigProps, IState> {
       <React.Fragment>
         <div className="config-background" />
         <div className="config-body">
-          <Typography>
+          <React.Fragment>
```

Comment:
I think you can remove the Fragments in this file?

Reply:
Yep!  Removed.